parameters/boolean_parameter.py

```diff
@@ -1,4 +1,3 @@
-# coding=utf-8
```

Comment:
No we don't need the this comment anymore?

Reply:
It's UTF8 by default in Python 3.
https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python3

Reply:
Ah I see...
I should follow your porting process so that I can also get new knowledge of python 3 :)

Thanks

Reply:
It's the big change in Py3, UTF8 by default and unicode too. No more encoding problem ;-)